templates/geosafe/analysis/create.html

```diff
@@ -411,19 +416,29 @@
     {% include "geosafe/analysis/modal/impact_card.html" %}
     {% include "geosafe/analysis/modal/iframe_modal.html" %}
     <div id="map" style="width: 100%; height: 100%; position:relative; margin-left: 70px; ">
-        <div class="legend">
+        <div class="legend" onmouseover="set_map_dragging(false);" onmouseout="set_map_dragging(true);">
             <div id="legend-title">
-                <a href="#"><H5 style="text-align: center">Legend</H5></a>
+                <a href="#" style="font-style: oblique;"><p>Legend</p></a>
             </div>
             <div id="legend-body">
-                <H5>Hazard</H5>
-                <p><input type="text" value="5" id="hazard-slider" data-slider-min="1" data-slider-max="10"
-                          data-slider-step="1" onchange="set_transparency();"></p>
-                <p><img id="hazard-legend-img"/></p>
-                <H5>Exposure</H5>
-                <p><input type="text" value="5" id="exposure-slider" data-slider-min="1" data-slider-max="10"
-                          data-slider-step="1" onchange="set_transparency();"></p>
-                <p><img id="exposure-legend-img"/></p>
+                <div id="legend-hazard-body">
+                    <p>Hazard</p>
+                    <p><input type="text" value="5" id="hazard-slider" data-slider-min="1" data-slider-max="10"
```

Comment:
yes, that's true.
1 to 10 scales serve for the UI  as the sliders use this range, instead of 0.0 to 1.0.
That's why the value will be divided by 10 later in the JS code.